Sources/Login/Views/InstitutionSelectionView.swift

```diff
@@ -73,6 +73,7 @@ private struct CustomInstanceCell: View {
 
             TextField(R.string.localizable.account_select_artemis_instance_custom_instance(), text: $customUrl)
                 .textFieldStyle(ArtemisTextField())
+                .keyboardType(.URL)
```

Comment:
@maximiliansoelch I just discovered your issue (https://github.com/ls1intum/artemis-ios-core-modules/issues/5). Anyway, I just changed it, it's merged.